src/Symfony/Component/Form/Tests/ChoiceList/ArrayChoiceListTest.php

```diff
@@ -18,13 +18,9 @@
  */
 class ArrayChoiceListTest extends AbstractChoiceListTest
 {
-    private $object;
-
     protected function setUp()
     {
         parent::setUp();
-
-        $this->object = new \stdClass();
```

Comment:
You should maybe just put it before the call to `parent::setUp()` instead of removing it and keep using `$this->object` in `getChoices()`